docs/single-bench.md

```diff
@@ -81,8 +81,6 @@ Notes:
 - After the site is ready the username is `Administrator` and the password is `$ADMIN_PASSWORD`
 - The local deployment is for testing and REST API development purpose only
 - A complete development environment is available [here](../development)
-- The site names are limited to patterns matching \*.localhost by default
-- Additional site name patterns can be added by editing /etc/hosts of your host machine
```

Comment:
this bit is not correct either. Name resolution can be re-configured by editing a few files including /etc/hosts. This will allow for other domain names in a purely local setup.